osu.Game.Rulesets.Osu/Mods/OsuModWiggle.cs

```diff
@@ -22,9 +24,13 @@ internal class OsuModWiggle : ModWithVisibilityAdjustment
         public override double ScoreMultiplier => 1;
         public override Type[] IncompatibleMods => new[] { typeof(OsuModTransform) };
 
-        private const int wiggle_duration = 90; // (ms) Higher = fewer wiggles
```

Comment:
Is there a reason this is preferable? Making it more understandable perhaps? I considered it but went with inlining to compact it

Reply:
It's used in multiple places. So if you wanted to change the duration for whatever reason you currently have to find all `90` literals rather than just touch the const.

Reply:
This does make sense. Will undo